index.html

```diff
@@ -6,12 +6,14 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <link rel="preconnect" href="https://fonts.googleapis.com" />
     <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
-    <link rel="preconnect" href="https://fonts.googleapis.com" />
-    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
     <link
       href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Roboto&display=swap"
       rel="stylesheet"
     />
```

Comment:
I think it is better to close the link tag in the same line on which you ended the text.  Kindly, please move "/>" to the previous line i-e line#11.  By doing this, your code will look more professional.